lib/livebook/live_markdown/import.ex

```diff
@@ -181,6 +193,20 @@ defmodule Livebook.LiveMarkdown.Import do
     build_notebook(elems, [cell | cells], sections)
   end
 
+  defp build_notebook([{:cell, :input, data} | elems], cells, sections) do
+    {metadata, elems} = grab_metadata(elems)
+
+    cell = %{
+      Notebook.Cell.new(:input)
+      | metadata: metadata,
+        type: data["type"] |> String.to_existing_atom(),
+        name: data["name"],
+        value: data["value"]
+    }
```

Comment:
What do you think about encapsulating this in Notebook.Cell.Input.from_json or similar?

Reply:
It's heavily related to livemarkdown, so keeping everything encapsulated in the import/export modules feels clearer to me, and it seems unlikely we will have a ton of such custom objects, because we prefer markdown whenever possible :)